tests/simulator/test_advance.py

```
@@ -378,9 +378,8 @@ def makerec(
                         elif box.ndim == 2:
                             print(f"failing cells: {failed_i}, {failed_j}")
                         print(coarsest_time)
-                        # if self.rethrow_:
-                        #     raise e
-                        # return diff_boxes(slice1, slice2, box)
+                        if self.rethrow_:
+                            raise e
```

Ruff (0.15.1)

[warning] Use `raise` without specifying exception name

Remove exception name

(TRY201)